src/sass/_base.scss

```
@@ -0,0 +1,146 @@
+html {
+    box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+body {
+    color: $color-text-body;
+    font-family: "Commissioner", sans-serif;
+    font-weight: 400;
+    margin: 0;
+    height: 100vh;
```

Setting a height like this on the `body` element can come back and create issues if you ever create a page that has text longer than anticipated. I know in this design you've handled that by having a separate element that scrolls, but you can also cover all your bases by updating this to be `min-height` instead.
```suggestion
    min-height: 100vh;
```